iis-wf/iis-wf-export-actionmanager/src/test/java/eu/dnetlib/iis/wf/export/actionmanager/sequencefile/FieldDecoder.java

```diff
@@ -0,0 +1,26 @@
+package eu.dnetlib.iis.wf.export.actionmanager.sequencefile;
+
+/**
+ * Field decoder interface.
+ * @author mhorst
+ *
+ * @param <T>
+ */
+public interface FieldDecoder {
+
+	/**
+	 * Decodes given source object into target representation.
+	 * @param source
+	 * @return decoded object
+	 * @throws Exception
+	 */
+	public Object decode(Object source) throws Exception;
+	
+	/**
+	 * Checks whether decoder can handle given input.
+	 * @param source
+	 * @return true when decoder can handle given input, false otherwise
+	 */
+	public boolean canHandle(Object source);
```

Comment:
Kind of. The initial intention was to allow using different decoder for the same `fieldName` and different type/context. 

This feature is not fully exploited by current `FieldAccessor` implementation yet.